### PR TITLE
Rewards details modal [WAIV-985]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/Details/Details.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Checkbox } from 'antd';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import detailsData from './detailsData';
+import './Details.less';
```
Comment (milklim) [inline src/client/rewards/Details/Details.js]: move to last line of imports

File: src/client/rewards/MatchBot/MatchBot.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ const MatchBot = ({ intl, userName }) => {
     if (userName) {
       getMatchBotRules(userName).then(data => {
         setRules(data);
-        setMinVotingPower(80);
+        setMinVotingPower(80); // change setting value when backend will ready
```
Comment (milklim) [inline src/client/rewards/MatchBot/MatchBot.js]: add "todo: " in comment

File: src/client/rewards/Details/Details.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Checkbox } from 'antd';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import detailsData from './detailsData';
```
Comment (milklim) [inline src/client/rewards/Details/Details.js]: import detailsData from './detailsData';
change to:
import getDetailsMessages from './detailsMsgData';

Comment (konstantin1390) [reply src/client/rewards/Details/Details.js]: done

File: src/client/rewards/Details/Details.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/Details/Details.js]: done

File: src/client/rewards/MatchBot/MatchBot.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/MatchBot/MatchBot.js]: done

**Pull request merged**